subprojects/build-cache-http/src/integTest/groovy/org/gradle/caching/http/internal/HttpBuildCacheServiceIntegrationTest.groovy

```diff
@@ -18,9 +18,7 @@ package org.gradle.caching.http.internal
 
 import org.gradle.integtests.fixtures.AbstractIntegrationSpec
 import org.gradle.test.fixtures.keystore.TestKeyStore
-import spock.lang.Timeout
 
-@Timeout(120)
```

Comment:
Can we keep the lower timeouts where established? I guess a class annotation is more specific than a superclass annotation, right?

IOW, I would expect to find `@IntegrationTestTimeout(120)` here. Same for the other locations.

Reply:
Makes sense. I'll change.